src/commands/forget.js

```diff
@@ -42,15 +44,29 @@ export default {
 		if (length === 0) return await interaction.editReply({ content: toSay });
 
 		try {
-			await interaction?.editReply({
-				content: toSay,
-				files: [
-					{
-						attachment: log,
-						name: "context.md",
+			const request = await fetch(
```

Comment:
A fallback error handler needs to be made here; the bot does catch the error, but we should be implementing some sort of intermediate `catch()` handler on this function to properly handle a case where the webserver is unreachable.